src/main/java/com/ebicep/warlords/player/general/SkillBoosts.java

```diff
@@ -146,41 +149,41 @@ public enum SkillBoosts {
             }
     ),
     WATER_BOLT("Water Bolt",
-            Component.text("Increase the healing of Water Bolt by 20%.", NamedTextColor.GRAY),
-            Component.text("Increase the healing of Water Bolt by ", NamedTextColor.GREEN)
-                     .append(Component.text("20%", NamedTextColor.RED))
+            Component.text("Increase the direct hit healing bonus of Water Bolt by 35%.", NamedTextColor.GRAY),
+            Component.text("Increase the direct hit healing bonus of Water Bolt by ", NamedTextColor.GREEN)
+                     .append(Component.text("35%", NamedTextColor.RED))
                      .append(Component.text(".", NamedTextColor.GREEN)),
             WaterBolt.class,
             abstractAbility -> {
-                if (abstractAbility instanceof WaterBolt) {
-                    abstractAbility.multiplyMinMax(1.2f);
+                if (abstractAbility instanceof WaterBolt waterBolt) {
+                    waterBolt.setDirectHitMultiplier(waterBolt.getDirectHitMultiplier() + 0.35);
                 }
             }
     ),
     WATER_BREATH("Water Breath",
-            Component.text("Increase the healing of Water Breath by 15% and reduce the energy cost by 30.", NamedTextColor.GRAY),
-            Component.text("Increase the healing of Water Breath by ", NamedTextColor.GREEN)
+            Component.text("Reduce the cooldown of Water Breath by 15% and reduce the energy cost by 30.", NamedTextColor.GRAY),
+            Component.text("Reduce the cooldown of Water Breath by ", NamedTextColor.GREEN)
                      .append(Component.text("15% ", NamedTextColor.RED))
                      .append(Component.text("and reduce the energy cost by ", NamedTextColor.GREEN))
                      .append(Component.text("30", NamedTextColor.RED))
                      .append(Component.text(".", NamedTextColor.GREEN)),
             WaterBreath.class,
             abstractAbility -> {
                 if (abstractAbility instanceof WaterBreath) {
-                    abstractAbility.multiplyMinMax(1.15f);
+                    abstractAbility.setCooldown(abstractAbility.getCooldown() * .15f);
                     abstractAbility.setEnergyCost(abstractAbility.getEnergyCost() - 30);
                 }
             }
     ),
     TIME_WARP_AQUAMANCER("Time Warp",
-            Component.text("Reduce the cooldown of Time Warp by 50%.", NamedTextColor.GRAY),
-            Component.text("Reduce the cooldown of Time Warp by ", NamedTextColor.GREEN)
-                     .append(Component.text("50%", NamedTextColor.RED))
+            Component.text("Increase the duration of Time Warp by 100%.", NamedTextColor.GRAY),
```

Comment:
Breaking like half the game by allowing people to warp across the map or something?

Reply:
No one uses Time Warp boost on Aqua because Aqua Time Warp is one of the most useless abilities in the game. This allows utility for warping flag and also leaving carrier to help team while guaranteeing coming back. Trade off is lost healing from other sbs